go.mod

```diff
@@ -1,3 +1,5 @@
-module github.com/redsift/dkim
+module dkim
```

Comment:
I made this change to the module name. 
But I can't remember doing it or why.

And it looks wrong to me.